package.json

```diff
@@ -162,7 +162,34 @@
       "testRule": true
     },
     "rules": {
-      "no-mixed-spaces-and-tabs": 0
+      "array-callback-return": "error",
+      "dot-notation": "error",
+      "func-name-matching": "error",
+      "guard-for-in": "error",
+      "no-else-return": [
+        "error",
+        {
+          "allowElseIf": false
+        }
+      ],
+      "no-implicit-coercion": "error",
+      "no-lonely-if": "error",
+      "no-mixed-spaces-and-tabs": "off",
+      "no-unneeded-ternary": "error",
+      "prefer-arrow-callback": "off",
+      "prefer-const": "error",
```

Comment:
We have it in main config.

And, frankly, it should be disabled, in my opinion.

Reply:
I'll remove it since it's the main config. I don't want to go into the trendy discussion about const vs let myself.